app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/tab/ProfileDestination.kt

```diff
@@ -160,13 +162,13 @@ private fun ProfileScreen(
         )
       }
       Spacer(Modifier.height(16.dp))
-      ProfileItemRows(
-        profileUiState = uiState,
-        showMyInfo = navigateToMyInfo,
-        showSettings = navigateToSettings,
-        showAboutApp = navigateToAboutApp,
-        navigateToEurobonus = navigateToEurobonus,
-        navigateToTravelCertificate = navigateToTravelCertificate,
+      ProfileRowsWithPlaceholders(
+        uiState,
+        navigateToMyInfo,
+        navigateToSettings,
+        navigateToAboutApp,
+        navigateToEurobonus,
+        navigateToTravelCertificate,
       )
```

Comment:
Since all of those here have the same signature of just `() -> Unit` it's super easy to flip one by accident and there won't be anything that would indicate this mistake.
Having the named parameter at least would make it more obvious if we did for example `showSettings = navigateToAboutApp,` to spot such problems. So I think we should make sure to add the named parameters back here.